src/components/App.tsx

```diff
@@ -370,7 +369,6 @@ export class App extends React.Component<any, AppState> {
     document.addEventListener("copy", this.onCopy);
     document.addEventListener("paste", this.pasteFromClipboard);
     document.addEventListener("cut", this.onCut);
-    window.addEventListener("touchstart", this.onTapStart);
```

Comment:
This shouldn't be done on the window but on the canvas. It also triggers when tapping multiple times on the UI.

Reply:
🥶. I’ll take a look